packages/mui-material-next/src/styles/Theme.types.ts

```diff
@@ -129,21 +129,35 @@ export interface MD3Typescale {
   display: TypescaleValue;
 }
 
-export interface Shapes {
-  borderRadius: number;
+export interface MD3Shape {
+  corner: {
+    none: string;
+    extraSmall: string;
```

Comment:
```suggestion
    extra-small: string;
```
Based on https://m3.material.io/styles/shape/shape-scale-tokens, I wonder if it should be a kebab case to follow the spec.

Reply:
Ah, the spec seems impossible with the:
```js
md.sys.shape.corner.extra-small
md.sys.shape.corner.extra-small.top
```